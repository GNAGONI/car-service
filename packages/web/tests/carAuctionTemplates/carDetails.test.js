import React from 'react'
import renderer from 'react-test-renderer'
import CarDetails from 'components/carAuctionTemplates/carDetails'

it('renders correctly', () => {
  const tree = renderer.create(<CarDetails />).toJSON()
  expect(tree).toMatchSnapshot()
})
