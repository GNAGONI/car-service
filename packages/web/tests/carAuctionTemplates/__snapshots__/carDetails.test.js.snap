// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="info-box"
>
  <h2>
    Car Details:
  </h2>
  <ul
    className="details-list"
  >
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="Has Keys"
          height={56}
          src=""
          width={54}
        />
      </span>
      <span
        className="text"
      >
        Has not Keys
      </span>
    </li>
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="Does Start"
          height={52}
          src=""
          width={54}
        />
      </span>
      <span
        className="text"
      >
        Does not Start
      </span>
    </li>
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="V5C Logbook"
          height={44}
          src=""
          width={46}
        />
      </span>
      <span
        className="text"
      >
        No V5
      </span>
    </li>
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="ABI Category"
          height={29}
          src=""
          width={62}
        />
      </span>
      <span
        className="text"
      >
        ABI category not recorded
      </span>
    </li>
    <li>
      <span
        className="icon-wrap"
      >
        <img
          alt="FSH"
          height={49}
          src=""
          width={36}
        />
      </span>
      <span
        className="text"
      >
        No FSH
      </span>
    </li>
  </ul>
</div>
`;
